secret_number.js

```diff
@@ -1,4 +1,10 @@
 'use strict';
 module.exports = function() {
-  
-};
\ No newline at end of file
+    let numbers = [];
+    return function secretNumber(){
+        if (numbers.length === 0){
```

Comment:
Although this works, it's best to set the number in the outer function instead. Then the inner function becomes a quick way to just retrieve the secret number.